app/javascript/react/components/ShowContainer.js

```diff
@@ -2,7 +2,7 @@ import React, { useState, useEffect } from "react"
 import ReviewContainer from "./ReviewContainer"
 
 const ShowContainer = (props) => {
-  const [movie, setMovie] = useState({})
+  const [movie, setMovie] = useState({ current_user: "" })
```

Comment:
we would expect that `current_user` is going to be an object, not a string for the initial state